cmd/client/main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"log"
+
+	pb "github.com/kubernetes/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+)
+
+const (
+	address = "localhost:49999"
+)
+
+func main() {
+	// Set up a connection to the server.
+	conn, err := grpc.Dial(address, grpc.WithInsecure())
+	if err != nil {
+		log.Fatalf("did not connect: %v", err)
+	}
+	defer conn.Close()
+	c := pb.NewRuntimeServiceClient(conn)
+
+	// Test RuntimeServiceClient.Version
+	version := "715fec664d75c6b5cb5b12718458621d4b75df37"
+	r, err := c.Version(context.Background(), &pb.VersionRequest{Version: &version})
```

Comment:
We should accept an argument to invoke this API.
For e.g. ./ociclient --runtimeversion